specification.json

```diff
@@ -230,7 +230,14 @@
         {
             "id": "Requirement 1.6.1",
             "machine_id": "requirement_1_6_1",
-            "content": "The API MUST define a `shutdown` function, which, when called, must call the respective `shutdown` function on the active provider.",
+            "content": "The API MUST define a `shutdown` function which, when called, must call the respective `shutdown` function on the active provider.",
+            "RFC 2119 keyword": "MUST",
+            "children": []
+        },
+        {
+            "id": "Requirement 1.6.2",
+            "machine_id": "requirement_1_6_2",
+            "content": "The client MUST define a `shutdown` function which, when called, removes all the event handlers associated with that client.",
```

Comment:
I think we should also define the client's behavior after the shutdown. I assume the client will act as backed by a noop provider.

Reply:
It's a minor point, but I'm a bit unsure about this now. I think we have 3 options:

- keep this as is, and add specific directions about "closed clients"
- keep this as is, and say that the behavior of closed client is undefined otherwise
- remove the client close point, and instead describe that handlers can be removed

Reply:
I am leaning towards the `remove the client close point, and instead describe that handlers can be removed` option. IMO it's too complex and unnecessary to spec out `closed clients`. But I am open for opinion of others here :)

Reply:
@Kavindu-Dodan I've made such a change. See [delta](https://github.com/open-feature/spec/pull/182/commits/a4e70ab3e89bdb23e145cd73e14bdce11022417c). I've kept it pretty vague since different languages might have different patterns for removing listeners/observers. The important part is they need to be able to be removed/deactivated, IMO.

Reply:
I verified the delta and I am satisfied with the change. Yes, agree with the fact that implementation details can vary, but spec must define the requirement :) 

Will resolve this conversation.